README.md

```diff
@@ -22,3 +13,11 @@ starttls-policy-cli --update-only [--policy-dir /path/to/dir]
 
 We currently only support Postfix, but contributions are welcome!
 
+## Development
+
+We recommend using `virtualenv` and `pip` to install and run `starttls-policy-cli` while developing. To get set up:
```

Comment:
And here it's still hyphens!

Reply:
Here `starttls-policy-cli` is name of `console_scripts` entry-point, defined in [setup.py](https://github.com/EFForg/starttls-policy-cli/blob/master/setup.py#L65). It's a different term and this name conforms to usual command line tool naming.